src/components/Dashboard/Details/tests/DetailItem.test.js

```diff
@@ -4,11 +4,19 @@ import { render } from 'enzyme';
 import { DetailItem } from '../DetailItem';
 import { default as DetailItemFixtures } from '../fixtures/DetailItem.fixture';
 
-const testDetailItem = () => <DetailItem {...DetailItemFixtures[0].props} />;
+const testDetailItem = index => <DetailItem {...DetailItemFixtures[index].props} />;
 
 describe('<DetailItem />', () => {
   it('renders correctly', () => {
-    const component = render(testDetailItem());
+    const component = render(testDetailItem(0));
```

Comment:
can you please render all the fixtures in for cycle and use their name as a test identitier?

Reply:
done